pandas/tests/io/formats/test_to_html.py

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8 -*-
 
 from datetime import datetime
-from io import open
+from io import StringIO, open
```

Comment:
Slightly orthogonal but shouldn't need import of open

Reply:
IIRC I think i added that to be able to read the expected output from a file in utf8 for the unicode test, xref #23747 because it was failing on py2. so can remove.